airflow_home/dags/dag.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+from datetime import datetime, timedelta
+
+import yaml
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+
+#  Need to make scripts package visible
+sys.path.append(os.environ.get('AIRFLOW_HOME'))
+
+from scripts.extract import extract
+from scripts.load import data_load
+from scripts.transform import find_result_data
+
+
+def load_config_from_yaml(file_path: str) -> dict():
+    with open(file_path, 'r') as f:
+        config = yaml.load(f, Loader=yaml.FullLoader)
+
+    return config
+
+
+default_args = {
+    'depends_on_past': False
+}
+
+
+def create_etl_tasks(
+    dag: DAG,
+    ticker: str,
+    task_before: PostgresOperator
+) -> None:
+    task_1 = PythonOperator(
+        task_id=f'extract_{ticker}',
+        python_callable=extract,
+        provide_context=True,
+        op_args=(ticker, ),
+        )
+
+    task_2 = PythonOperator(
+        task_id=f'transform_{ticker}',
+        python_callable=find_result_data,
+        provide_context=True,
+        op_args=(ticker, ),
+        )
+
+    task_3 = PythonOperator(
+        task_id=f'load_{ticker}',
+        python_callable=data_load,
+        provide_context=True,
+        op_args=(ticker, ),
+        )
+
+    task_before.set_downstream(task_1)
+    task_1.set_downstream(task_2)
+    task_2.set_downstream(task_3)
+
+
+with DAG(
+    'tickers_to_postgres',
+    default_args=default_args,
+    description='Tickers from API loads into PostgreSQL',
+    schedule_interval=timedelta(hours=2),
```

Comment:
A good way to use this `timedelta` functionality. This is pretty suitable for the workloads like that with repetitive time intervals. But remember to use crons If you don't have some well-defined interval.

Easy way to create and modify cron tabs: https://crontab.guru/